fdbserver/DDTxnProcessor.actor.cpp

```diff
@@ -901,10 +901,10 @@ void DDMockTxnProcessor::setupMockGlobalState(Reference<InitialDataDistribution>
 		}
 		// insert to serverKeys
 		for (auto& id : shardInfo.primarySrc) {
-			mgs->allServers[id].serverKeys.insert(keys, { MockShardStatus::COMPLETED, shardBytes });
+			mgs->allServers.at(id)->serverKeys.insert(keys, { MockShardStatus::COMPLETED, shardBytes });
 		}
 		for (auto& id : shardInfo.primaryDest) {
-			mgs->allServers[id].serverKeys.insert(keys, { MockShardStatus::INFLIGHT, shardBytes });
+			mgs->allServers.at(id)->serverKeys.insert(keys, { MockShardStatus::INFLIGHT, shardBytes });
```

Comment:
Are we guaranteed to find `id` in `allServers`? Previously `[]` would've inserted the element if it didn't exist, but `at()` will throw an exception.

Reply:
> Are we guaranteed to find id in allServers
It should be always true. If the exception thrown here, it indicates a bug.